.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Launching the app returns a blank page. User has to manually add /login at the end of the url to access the project.